### PR TITLE
[Docs] Add some introduction

### PR DESCRIPTION
**Pull request by xialovexia**
Add some introduction and crowdloan part about it. Still need some modification and language check.

### CODE REVIEW

**Comment by ghzlatarev** [comment]
You can branch this from main now. The other PR is merged.

File: docs/calamari/node-operators/crowdloan.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+
+
+`Crowdloan` allows a team to crowdsource KSM (for Kusama slots) or DOT (for Polkadot slots) to use to bootstrap its parachain auction. So crowdloan allows people to contribute by agreeing to lock up their own KSM (DOT) until the end of the lease. Teams can reward their contributors however they see fit, and can structure their crowdloan in various ways, hosting it either natively on Kusama (Polkadot) or on a 3rd-party platform. 
+
+## How to join ours ?
+
+### **Install polkadot.js/extension.**
+
+Use polkadot.js/extension, you can login in our crowdloan system easily. If you haven't already installed it, go to https://polkadot.js.org/extension/ and add this extension to your browser. And also you can find how to add a account to this extension.
+
```
Comment (ghzlatarev) [inline docs/calamari/node-operators/crowdloan.md:16]: an* account

File: docs/calamari/node-operators/crowdloan.md [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+If you ready you can see it: ![PJS-EX](crowdloan.assets/PJS-EX.png)
+
+### **Open our crowdloan page and login your account**
+
+After you prepare ready for your polkadot.js/extension, go and open our crowdloan front page https://crowdloan.calamari.manta.systems. The first time we you open this page, you need to allow our crowdloan system to connect with polladot.js/extension. If you have down, you can see this page:
```
Comment (ghzlatarev) [inline docs/calamari/node-operators/crowdloan.md:21]: ``After you prepared your poilkadot.js/extension`` sounds better.

File: docs/calamari/node-operators/crowdloan.md [hunk 3]
```diff
@@ -1,9 +1,34 @@
...
+
+![Calamari-crowdloan](crowdloan.assets/Calamari-crowdloan.png)
+
+Choose your language you want, and then click the bottom `Connect Wallet` to login your account in our crowdloan system.
```
Comment (ghzlatarev) [inline docs/calamari/node-operators/crowdloan.md:25]: ``Choose the language you want``

File: docs/calamari/node-operators/crowdloan.md [hunk 4]
```diff
@@ -1,9 +1,34 @@
...
+
+### **Claim your contribution**
+
+Your can start your contribute now. Enter your contribution amout in the first inputbox the enter your referral code you get from our Manta community, like discord, element chat room or twitter. Click `Claim your KMA` and accept this operator in your extension. ![contribution](crowdloan.assets/contribution.png)
```
Comment (ghzlatarev) [inline docs/calamari/node-operators/crowdloan.md:29]: ``You can start contributing now``

Comment (ghzlatarev) [reply docs/calamari/node-operators/crowdloan.md:29]: ``amount``

Comment (ghzlatarev) [reply docs/calamari/node-operators/crowdloan.md:29]: ``referral code you got``

File: docs/calamari/node-operators/crowdloan.md [hunk 5]
```diff
@@ -1,9 +1,34 @@
...
+
+After contribution processing finish (Under the `Claim your KMA` button, you can see a small line to show your contribution processing ), you can see your rewards and your contribution details now.
```
Comment (ghzlatarev) [inline docs/calamari/node-operators/crowdloan.md:31]: ``After the contribution process finishes (Under the `Claim your KMA` button, you can see a small line to show your contribution processing), you should see your rewards and contribution details.``

File: docs/calamari/node-operators/introduction.md [hunk 6]
```diff
@@ -6,4 +6,13 @@ sidebar_position: 1
...
+
+In early period, Calamari parachain will provide two products:
+* **MariPay**, its token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular assets like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.
+* **MariSwap**, its private AMM-based DEX. MariSwap offers users the capability of swaps between parachain assets while preserving the privacy of the user addresses. It also provides never-before-seen features in the industry such as private liquidity pools.
+
+Calamari will be the only platform on Kusama that can meet user on-chain privacy needs, boost anti-surveillance, and ensure private tokens’ decentralization and interoperability. Now Calamari we provide is a shell-chain.
+
```
Comment (ghzlatarev) [inline docs/calamari/node-operators/introduction.md:16]: ``Currently Calamari is a shell-chain`` I think we should start with that.

File: docs/calamari/node-operators/crowdloan.md (hunk 2 above)
Comment (xialovexia) [reply docs/calamari/node-operators/crowdloan.md:21]: Thanks~

File: docs/calamari/node-operators/introduction.md (hunk 6 above)
Comment (xialovexia) [reply docs/calamari/node-operators/introduction.md:16]: Yeah, I agree. But I am not very sure how to explain to shell chain

File: docs/calamari/introduction.md [hunk 7]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
 
 ## What is Calamari parachain
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:7]:
```suggestion
## The Calamari Parachain
```

File: docs/calamari/introduction.md [hunk 8]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
 
-### What is a shell-chain 
\ No newline at end of file
+Calamari, Manta Network’s canary-net (the canary-net is an early, highly experimental version of Manta Network, where all new features will be released and tested first), is the plug-and-play privacy-preservation parachain built to service the Kusama DeFi world. It combines Kusama and zkSNARKs to bring on-chain privacy to transactions and swaps. As part of its own suite of products, Calamari Network offers:
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:9]:
```suggestion
Calamari is Manta Network’s canary-net, an early, highly experimental version of Manta Network, where all new features will be released and tested first. Calamari is the plug-and-play privacy-preservation parachain built to service the Kusama DeFi world. It combines Kusama and zkSNARKs to bring on-chain privacy to transactions and swaps. As part of its own suite of products, Calamari Network offers:
```

File: docs/calamari/introduction.md [hunk 9]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
+
+In early period, Calamari parachain will provide two products:
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:11]: Choose either this sentence or the last one of the previous paragraph.

File: docs/calamari/introduction.md [hunk 10]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
+* **MariPay**, its token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular assets like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:12]:
```suggestion
* **MariPay:** a token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular assets like stablecoins and wrapped BTC while simultaneously getting the benefits of on-chain privacy through ZKP.
```

File: docs/calamari/introduction.md [hunk 11]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
+* **MariSwap**, its private AMM-based DEX. MariSwap offers users the capability of swaps between parachain assets while preserving the privacy of the user addresses. It also provides never-before-seen features in the industry such as private liquidity pools.
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:13]:
```suggestion
* **MariSwap:** a private AMM-based DEX. MariSwap offers users the capability of swaps between parachain assets while preserving the privacy of the user addresses. It also provides never-before-seen features in the industry such as private liquidity pools.
```

File: docs/calamari/introduction.md [hunk 12]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
+
+Calamari will be the only platform on Kusama that can meet user on-chain privacy needs, boost anti-surveillance, and ensure private tokens’ decentralization and interoperability. Currently Calamari is a shell-chain.
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:15]: Is this true? Will it always be true? Seems like you could say something a little less strong: 


```suggestion
Calamari will meet Kusama user's needs for on-chain privacy, anti-surveilance, and ensuring that private tokens are decetralized and interoperable. Calamari is currently a shell-chain.
```

File: docs/calamari/introduction.md [hunk 13]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
+
+### What is a shell-chain 
+
+The `shell-chain` is a `purely` parachain that it produces blocks, but has no functionality beyond being upgradable. It is our staging ground, and as the first common-good parachain. It has no concept of user accounts, staking, governance or balances. All it does is listen out for a message coming from the Relay Chain’s governance apparatus authorizing it to upgrade. When such a message is received, the Shell parachain will allow itself to be upgraded to whatever new logic the Relay Chain has authorized, and we call it `Runtime Upgrade`. Manta will upgrade Calamari with core features such as `MariPay` and `MariSwap` through runtime upgrade. And we will continue to improve them in subsequent upgrades.
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:19]:
```suggestion
A _shell-chain_ is a _pure_ parachain that produces blocks but has no functionality beyond being upgradable. It is our staging ground, and the first common-good parachain. It has no concept of user accounts, staking, governance or balances. All it does is listen out for a message coming from the Relay Chain’s governance apparatus authorizing it to upgrade. When such a message is received, the Shell parachain will allow itself to be upgraded to whatever new logic the Relay Chain has authorized, and we call it `Runtime Upgrade`. The Manta development team will upgrade Calamari with core features such as `MariPay` and `MariSwap` through runtime upgrade, continue to improve the functionality in subsequent upgrades.
```

File: docs/calamari/introduction.md [hunk 14]
```diff
@@ -6,4 +6,16 @@ sidebar_position: 1
...
+
+There are some information about our Calamari. Calamari's parachain ID on Kusama is 2084. and our SS58 prefix is 78. Our total issuance of $KMA is 10B. $KMA is the first community-driven deflationary utility token where rebate rewards, redemption rewards, and token burns are generated from the Calamari privacy-preserving network usage. The Initial Distribution of KMA is up to 10 billion tokens. There will be no new token generation unless KMAs for the crowdloan are consumed.
```
Comment (gabcbrown) [inline docs/calamari/introduction.md:21]:
```suggestion
## Calamari Chain Information

Calamari's parachain ID on Kusama is 2084. and our SS58 prefix is 78. Our total issuance of $KMA is 10B. $KMA is the first community-driven deflationary utility token where rebate rewards, redemption rewards, and token burns are generated from the Calamari privacy-preserving network usage. The Initial Distribution of KMA is up to 10 billion tokens. There will be no new token generation unless KMAs for the crowdloan are consumed. 
```

**Review by gabcbrown — CHANGES_REQUESTED** [review]

File: docs/calamari/crowdloan.md [hunk 15]
```diff
@@ -6,4 +6,41 @@ sidebar_position: 4
 
 ## What is it ?
 
-## How to join ours ?
\ No newline at end of file
+All Kusama parachains compete for the chance to launch a blockchain on Kusama by participating in a parachain slot auction. A parachain slot auction is when teams bid in an auction for the chance to win a slot to launch a blockchain on Kusama for security and interoperability. To help the team raise funds to participate in the auction, Kusama allows teams to launch a `crowdloan` in the community.
```
[the remark this answers is not in the document]
Comment (xialovexia) [reply docs/calamari/crowdloan.md:9]: Great! Thanks

File: docs/calamari/crowdloan.md [hunk 16]
```diff
@@ -6,4 +6,41 @@ sidebar_position: 4
...
+Use polkadot.js/extension, you can login in our crowdloan system easily. If you haven't already installed it, go to https://polkadot.js.org/extension/ and add this extension to your browser. And also you can find how to add an account to this extension.
+
+If you ready you can see it: ![PJS-EX](crowdloan.assets/PJS-EX.png)
+
+### **Open our crowdloan page and login your account**
+
+After you prepared your poilkadot.js/extension, go and open our crowdloan front page https://crowdloan.calamari.manta.systems. The first time we you open this page, you need to allow our crowdloan system to connect with polladot.js/extension. If you have down, you can see this page:
+
+If you ready you can see it: ![PJS-EX](crowdloan.assets/PJS-EX.png)
+
```
[the remark this answers is not in the document]
Comment (xialovexia) [reply docs/calamari/crowdloan.md:27]: My bad, remove it~

File: docs/calamari/introduction.md (hunk 12 above)
Comment (xialovexia) [reply docs/calamari/introduction.md:15]: Yeah, it refers from our Manta website news, but I think your expression is more reasonable

File: docs/calamari/introduction.md (hunk 14 above)
Comment (xialovexia) [reply docs/calamari/introduction.md]: Thanks for your suggestion, I have already added your suggestion to the new commits.

File: docs/calamari/crowdloan.md [hunk 17]
```diff
@@ -6,4 +6,41 @@ sidebar_position: 4
...
 
-## How to join ours ?
\ No newline at end of file
+All Kusama parachains compete for the chance to launch a blockchain on Kusama by participating in a parachain slot auction. A parachain slot auction is when teams bid in an auction for the chance to win a slot to launch a blockchain on Kusama for security and interoperability. To help the team raise funds to participate in the auction, Kusama allows teams to launch a `crowdloan` in the community.
+
+
+`Crowdloan` allows a team to crowdsource KSM (for Kusama slots) or DOT (for Polkadot slots) to use to bootstrap its parachain auction. So crowdloan allows people to contribute by agreeing to lock up their own KSM (DOT) until the end of the lease. Teams can reward their contributors however they see fit, and can structure their crowdloan in various ways, hosting it either natively on Kusama (Polkadot) or on a 3rd-party platform. 
+
+## How to join ours ?
+
```
[the remark this answers is not in the document]
Comment (stechu) [reply docs/calamari/crowdloan.md:15]: @xialovexia

File: docs/calamari/crowdloan.md [hunk 18]
```diff
@@ -4,6 +4,38 @@ sidebar_position: 4
...
+## Details about Calamari Crowdloan
+
+Calamari Network intends to begin its Parachain Loan Offering in conjunction with the start of the sixth Kusama parachain auction. The exact date marking the commencement of the sixth auction is unclear at the time of writing; the fifth auction is finishing up now, and will end on Friday, July 23, 2021. After that auction completes, there will be a review period before beginning the second batch of auctions (auctions 6 through 10). At the moment, the duration of the review period is unclear.
+
+Anyone who participate in our Calamari PLO will reward you with $KMA, which is Calamari’s native token. Calamari crowdloan contributors will receive 10,000 KMAs per 1 KSM contributed as rewards for their contribution to the crowdloan by default. The fixed ratio guarantees that rewards for KSM contributors won’t change with the amount of KSM Calamari raised. This makes it fair for all participants, regardless of the when they enter. Also we are offering early bonuses to the first 1,000 contributors on our crowdloan. For more infomation, please go and see it: https://mantanetwork.medium.com/the-calamari-crowdloan-on-kusama-74a3cb2a2a4b.
```
Comment (gabcbrown) [inline docs/calamari/crowdloan.md:18]:
```suggestion
Anyone who participates in our Calamari PLO will be rewarded with $KMA, Calamari’s native token. Contributors will receive 10,000 KMAs per 1 KSM contributed. The fixed ratio guarantees that rewards for KSM contributors won’t change with the amount of KSM Calamari raised. This makes it fair for all participants, regardless of the when they enter. Also we are offering early bonuses to the first 1,000 contributors on our crowdloan. For more infomation, please read: https://mantanetwork.medium.com/the-calamari-crowdloan-on-kusama-74a3cb2a2a4b.
```

File: docs/calamari/crowdloan.md [hunk 19]
```diff
@@ -4,6 +4,38 @@ sidebar_position: 4
...
+
+## How to join the Calamari Crowdloan ?
+
+### **Install polkadot.js/extension.**
+
+Using the polkadot.js/extension you can login in our crowdloan system easily. If you haven't already installed it, go to https://polkadot.js.org/extension/ and add this extension to your browser. The website also describes how to add an account to this extension.
```
Comment (gabcbrown) [inline docs/calamari/crowdloan.md:24]:
```suggestion
Using the polkadot.js/extension you can login into our crowdloan system easily. If you haven't already installed it, go to https://polkadot.js.org/extension/ and add the extension to your browser, and then add an account to the extension.
```

File: docs/calamari/crowdloan.md [hunk 20]
```diff
@@ -4,6 +4,38 @@ sidebar_position: 4
...
+
+You will see your accounts show up like this: ![PJS-EX](crowdloan.assets/PJS-EX.png)
+
+### **Open our crowdloan page and login your account**
+
+After you've set up your poilkadot.js/extension, go to the crowdloan page: https://crowdloan.calamari.manta.systems. The first time you open this page, you'll need to allow our crowdloan system to connect with the polladot.js/extension. Once it's connected to your account, you'll see:
```
Comment (gabcbrown) [inline docs/calamari/crowdloan.md:30]:
```suggestion
After you've set up your polkadot.js/extension, go to the crowdloan page: https://crowdloan.calamari.manta.systems. The first time you open this page, you'll need to allow our crowdloan system to connect with the polladot.js/extension. Once it's connected to your account, you'll see:
```

File: docs/calamari/crowdloan.md [hunk 21]
```diff
@@ -4,6 +4,38 @@ sidebar_position: 4
...
+
+If you ready you can see it: ![PJS-EX](crowdloan.assets/PJS-EX.png)
```
Comment (gabcbrown) [inline docs/calamari/crowdloan.md:32]:
```suggestion
![PJS-EX](crowdloan.assets/PJS-EX.png)
```

**Review by gabcbrown — CHANGES_REQUESTED** [review]

**Review by gabcbrown — APPROVED** [review]

**Review by stechu — APPROVED** [review]

**Pull request merged**